resources/forwarding_rule.rb

```diff
@@ -205,6 +205,9 @@ def update
             # TODO(nelsonjr): Check w/ Chef... can we print this in red?
             puts # making a newline until we find a better way TODO: find!
             compute_changes.each { |log| puts "    - #{log.strip}\n" }
+              if (@current_resource.target != @new_resource.target)
```

Comment:
Some of the indentation in this file looks off - is that just the view GitHub has?